src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx

```diff
@@ -170,7 +170,8 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
         <Formik
             enableReinitialize
             initialValues={fields}
-            //validationSchema={validationSchema}
+            validationSchema={validationSchema}
+            validateOnBlur={false}
```

Comment:
isn't it this option we discussed yesterday ?

Reply:
Yes. I have started yup implementation. Only two fields are required a this Time.